src/BoilerPlate/Asteroids.cpp

```diff
@@ -0,0 +1,76 @@
+#include "Asteroids.hpp"
+const float SPACE_MAGIC = 30.0f;
+const float MAX_SPEED = 60.0f;
+const float ASTEROID_RADIO = 27.0f;
+
+Asteroids::Asteroids() {}
+Asteroids::Asteroids(float width,float height):
+	Entity(width,height) {
+
+	asteroid_Size = 1;
+	angle = 0;
+	PushDrawEntity();
+	mass = asteroid_Size;
+	position = Vector2(rand(),rand());
+	velocity = Vector2(0, 0);
+}
+Asteroids::Asteroids(Asteroid_Size size, float width, float height):
```

Comment:
If you have a Game representation width and height should not be passed to entities